### PR TITLE
Upgrade to CentOS 9

### PR DESCRIPTION
**Pull request by mikedep333**
Includes postgres upgrade logic from 12 to 13

### CODE REVIEW

File: assets/init/postgres-init [hunk 1]
```diff
@@ -13,16 +13,21 @@ export C035="\e[35m"
 export C036="\e[36m"
 export C037="\e[37m"
 
-PGVERSION=$(postgres --version | egrep -o "[0-9]{1,}\.[0-9]{1,}")
+PGVERSION=$(postgres --version | egrep -o "1[0-9]")
```
Comment (mdellweg) [inline assets/init/postgres-init:16]: Why is the old regex wrong? What is this supposed to match?

It looks quite different to me:
```
[root@f7c7ec457e7a /]# postgres --version | egrep -o "1[0-9]"
12
11
[root@f7c7ec457e7a /]# postgres --version | egrep -o "[0-9]{1,}\.[0-9]{1,}"
12.11
```

**Review by mdellweg — COMMENTED** [review]
Very nice that we care about the upgrade now!

File: pulp_galaxy_ng/Containerfile [hunk 2]
```diff
@@ -8,6 +8,6 @@ RUN pip3 install --upgrade \
   galaxy-ng${GALAXY_NG_VERSION} && \
   rm -rf /root/.cache/pip
 
-RUN ln /usr/local/lib/python3.8/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_ansible.conf
```
Comment (fao89) [inline pulp_galaxy_ng/Containerfile:11]: this is the only concern I have, we'll stop to test pulp on python 3.8

File: assets/init/postgres-init (hunk 1 above)
Comment (mikedep333) [reply assets/init/postgres-init:16]: The upgrade command is based on the major version of postgresql, not the minor version anymore. The minor version was relevant for 9.x I think.

File: pulp_galaxy_ng/Containerfile (hunk 2 above)
Comment (mikedep333) [reply pulp_galaxy_ng/Containerfile:11]: Yeah, there is no way around this. Unless we want to maintain 2 different images, which is not ideal.

I will close this PR but not delete my branch. We will implement this later.

**Pull request closed without merging**